.github/workflows/testsuite.yml

```diff
@@ -36,8 +36,9 @@ jobs:
     strategy:
       matrix:
         meteorRelease:
-          - '2.8.0'
-          - '2.14'
+          - '2.8.1'
+          - '2.16'
+          - '3.0'
```

Comment:
What's the use of testing on older versions if we're going to introduce breaking changes? I think it'd be best to only have 3 and up.

Reply:
We are technically backward compatible to 2.8.1.